manifests/pagerduty.pp

```diff
@@ -2,16 +2,23 @@
 #
 # Sensu handler for communicating with Pagerduty
 #
-class sensu_handlers::pagerduty inherits sensu_handlers {
+class sensu_handlers::pagerduty (
+  $pagerduty_package       = 'redphone',
+  $pagerduty_package_opts  = { 'provider' => 'gem' }
```

Comment:
How about defaulting to sensu_gem?

I guess that would mean that we would have to change the #! on the handler at the same time?

To prepare for this at yelp we can install both I guess and then make the cutover